### PR TITLE
New footer with github and fb links

### PR DESCRIPTION
**Pull request by icanberk**

### CODE REVIEW

**Comment by ymzong** [comment]
:+1:

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -27,6 +27,15 @@
...
 
   <body class="autolab">
+    <div id="fb-root"></div>
+    <script>(function(d, s, id) {
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.3&appId=259908887444831";
+      fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));</script>
```
Comment (dlbucci) [inline app/views/layouts/application.html.erb]: What is this code doing?

Comment (jez) [inline app/views/layouts/application.html.erb]: Looks like it adds this:

screenshot 2015-04-25 21 52 31

Comment (dlbucci) [inline app/views/layouts/application.html.erb]: Ah, okay, I guess that's what 'fb link' would probably mean...

Comment (jez) [inline app/views/layouts/application.html.erb]: Since it's JavaScript, it should be in its own file though, not in application.html.erb

Comment (dlbucci) [inline app/views/layouts/application.html.erb]: I'm actually okay with this, seeing as it's one less file to load.  I would put it at the bottom of the body tag, or if you're going to move it into a file, make it a file that contains code for every dependency (github and facebook and heap, etc.).

**Pull request merged**